webui/react/src/stores/users.tsx

```diff
@@ -45,7 +45,9 @@ export const UsersProvider: React.FC<PropsWithChildren> = ({ children }) => {
   );
 };
 
-export const useFetchUsers = (canceler: AbortController): (() => Promise<void>) => {
+export const useFetchUsers = (
+  canceler: AbortController,
+): ((cfg?: FetchUsersConfig) => Promise<DetailedUserList | void>) => {
```

Comment:
i think return type should be only one

Reply:
`fetchUsers` should just return `void`. Fetch functions should always only return void and if you want to subscribe to the value changing, you should do a `useEffect` on the actual value. That's a fundamental tenant of reactive programming.